lib/3scale_toolbox/commands/service_command/service_remove.rb

```diff
@@ -0,0 +1,32 @@
+module ThreeScaleToolbox
+  module Commands
+    module ServiceCommand
+      class ServiceRemoveSubcommand < Cri::CommandRunner
+        include ThreeScaleToolbox::Command
+
+        def self.command
+          Cri::Command.define do
+            name        'remove'
+            usage       'remove'
+            summary     'service remove'
+            description 'Remove a service'
+            runner ServiceRemoveSubcommand
+
+            param   :source
+            param   :service_id
```

Comment:
Format of the command:

`3scale service remove <source> <service-id>`

Reply:
`<service-id_or_system-name>`